src/main/java/bookstore/controller/BookController.java

```diff
@@ -36,7 +37,7 @@ public BookDto getBookById(@PathVariable Long id) {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
```

Comment:
1) Let’s validate the request dto in the method update

2) Optional:
you may also validate such things as @PathVariable, @RequestParams etc. It looks like:

`@PathVariable @Positive Long id`
In order to be able to perform such validation you need to annotate your controller class with Validated annotation